swingset/src/main/java/com/nqadmin/swingset/SSDataNavigator.java

```diff
@@ -329,12 +352,22 @@ private void performUpdates() {
 	 */
 	protected JButton undoButton = new JButton();
 
+	//
+	// TODO:
+	//     For now, have the defaults here. In the future,
+	//     probably want to set the defaults from some
+	//     configurable spot, via CentralLookup?
+	//     Maybe: interface SwingSetConfiguration {}
+	//
+	private static final boolean V3_BUTTONS_DEFAULT = true;
+	private static final boolean AUTO_COMMIT_DEFAULT = false;
+
```

Comment:
We have swingset.properties in src/main/resources too.

Reply:
This is a placeholder, and a way to indicate that this is something different screens/apps/whatever might want to configure globally.